src/panels/lovelace/cards/hui-sensor-card.ts

```diff
@@ -257,6 +257,7 @@ class HuiSensorCard extends LitElement implements LovelaceCard {
       `;
     }
 
+    const graphColor = this._config.graph_color || "var(--accent-color)";
```

Comment:
I don't think we should add this yet. I think we should wait until we get a blanket option for this.

My last comment was to change the variable name not add a config variable

Reply:
@bramkragten What is the verdict? Creating a new CSS variable that falls back to Accent color and leaving this to themes for now or going ahead with the new config variable?

If we wait then we can add a theme tab to all cards with x options to change. Otherwise, why are we just adding it to the sensor card? Why not the gauge card too.

Reply:
I don't want to start with theme related config settings, if we start that we will never get out of it.

Reply:
Agree. So lets start with a new CSS variable that is targetable via a theme for the sensor card @timmo001 

This is great conversation and a good start but we need to do it right